include/m_home.h

```diff
@@ -29,34 +29,37 @@ typedef enum mHm_ROOMTYPE {
 } mHm_ROOMTYPE;
 
 typedef enum mHm_HOMESIZE {
-    /* 0 */ mHm_HOMESIZE_SMALL,   /* initial size */
-    /* 1 */ mHm_HOMESIZE_MEDIUM,  /* paid off first debt */
-    /* 2 */ mHm_HOMESIZE_LARGE,   /* paid off second debt (excluding basement) */
-    /* 3 */ mHm_HOMESIZE_UPPER,   /* paid off third debt & basement */
-    /* 4 */ mHm_HOMESIZE_STATUE,  /* paid off final debt */
+    /* 0 */ mHm_HOMESIZE_SMALL,   // initial size
+    /* 1 */ mHm_HOMESIZE_MEDIUM,  // paid off first debt
+    /* 2 */ mHm_HOMESIZE_LARGE,   // paid off second debt (excluding basement)
+    /* 3 */ mHm_HOMESIZE_UPPER,   // paid off third debt & basement
+    /* 4 */ mHm_HOMESIZE_STATUE,  // paid off final debt
 
     /* 5 */ mHm_HOMESIZE_NUM
 }mHm_HOMESIZE;
```

Comment:
```suggestion
}HomeHomeSize;
```

Maybe? I mean the struct name is a bit silly to be honest, but follows styling